src/gui/advancedsettings.cpp

```diff
@@ -137,6 +137,7 @@ namespace
         PEER_TURNOVER,
         PEER_TURNOVER_CUTOFF,
         PEER_TURNOVER_INTERVAL,
+        REANNOUNCE_WHEN_ADDRESS_CHANGED,
```

Comment:
Please move it above `DOWNLOAD_TRACKER_FAVICON`.
This includes moving other new code to the relative position.